src/name_bazaar/ui/components/offering/bottom_section.cljs

```diff
@@ -63,7 +72,9 @@
            :disabled (not editable?)
            :color "purple"
            :href (path-for :route.offerings/edit {:offering/address address})}
-          "Edit"])])))
+          "Edit"])
+       (when can-delete?
+         [delete-offering-button {:address address :offering-type type}])])))
```

Comment:
rename keys to `:offering/address`, `:offering/type` for consistency with rest of app